src/frequenz/client/dispatch/__main__.py

```diff
@@ -32,24 +32,22 @@
 )
 from ._client import Client
 
-DEFAULT_DISPATCH_API_HOST = "fz-0004.frequenz.io"
-DEFAULT_DISPATCH_API_PORT = 50051
+DEFAULT_DISPATCH_API_URL = "grpc://fz-0004.frequenz.io:50051?ssl=true"
 
 
-def get_client(*, host: str, port: int, key: str, insecure: bool) -> Client:
+def get_client(*, url: str, key: str, insecure: bool) -> Client:
     """Get a new client instance.
 
     Args:
-        host: The host of the dispatch service.
-        port: The port of the dispatch service.
+        url: The URL for the Dispatch API.
         key: The API key for authentication.
         insecure: Allow use of insecure connection.
 
     Returns:
         Client: A new client instance.
     """
     return Client(
-        server_url=f"grpc://{host}:{port}?ssl={not insecure}",
+        server_url=f"grpc://{url}?ssl={not insecure}",
```

Comment:
Is this right? Doesn't `url` also include the `grpc://` (and potentially a `?` with some other options)?

Why not just removing the `insecure: bool` option and passing that as part of the URL?